package.json

```diff
@@ -12,14 +12,23 @@
   "devDependencies": {
     "ava": "^0.22.0",
     "ava-spec": "^1.1.0",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
     "eslint": "^4.6.1",
     "eslint-config-mixmax": "^0.6.0",
     "nyc": "^11.2.1"
   },
+   "files": [
+    "index.js",
```

Comment:
What `index.js` file is it we need to put in the shipped package? Shouldn't it only be the `dist/` dir now?

Reply:
Ah yes, will fix